tracker/tracker.go

```diff
@@ -420,6 +435,13 @@ func (e *EventTracker) processLogs() error {
 			if log.Topics[0] == id {
 				filteredLogs = append(filteredLogs, log)
 
+				if e.config.EventSubscriber == nil {
```

Comment:
You can check if this is nil in the constructor function, and remove this from the processLogs.